tests/testthat/test-RunAnalyses.R

```diff
@@ -0,0 +1,38 @@
+library(testthat)
+
+test_that("multiple analyses", {
+  # Analysis.R is checked elsewhere
+  exposureOutcome1 <- createExposureOutcome(767410, 444382)
+  exposureOutcome2 <- createExposureOutcome(1314924, 444382)
+  exposureOutcome3 <- createExposureOutcome(907879, 444382)
+  exposureOutcomeList <- list(exposureOutcome1, exposureOutcome2, exposureOutcome3)
+
+  runSelfControlledCohortArgs1 <- createRunSelfControlledCohortArgs(firstExposureOnly = FALSE)
+  runSelfControlledCohortArgs2 <- createRunSelfControlledCohortArgs(firstExposureOnly = TRUE)
+  sccAnalysis1 <- createSccAnalysis(analysisId = 1,
+                                    runSelfControlledCohortArgs = runSelfControlledCohortArgs1)
+  sccAnalysis2 <- createSccAnalysis(analysisId = 2,
+                                    runSelfControlledCohortArgs = runSelfControlledCohortArgs2)
+  sccAnalysisList <- list(sccAnalysis1, sccAnalysis2)
+
+  outputFolder <- file.path(tempdir(), getOption("dbms"))
+  dir.create(outputFolder)
+  on.exit(unlink(outputFolder))
```

Comment:
Could you confirm that the `on.exit` clause gets executed at the end of the unit test? (I've only used it inside functions, so I'm not sure it works in this context)